src/nvim/highlight_group.c

```diff
@@ -230,61 +230,108 @@ static const char *highlight_init_both[] = {
   "default link DiagnosticSignOk           DiagnosticOk",
   "default link DiagnosticUnnecessary      Comment",
 
-  // Text
+  // Treesitter standard groups
+  "default link @variable            NONE",  // don't highlight to reduce visual overload
+  "default link @variable.builtin    Identifier",
+  "default link @variable.parameter  Identifier",
+  "default link @variable.member     Identifier",
+
+  "default link @constant          Constant",
+  "default link @constant.builtin  Special",
+  "default link @constant.macro    Define",
+
+  "default link @module  Module",
+  "default link @label   Label",
+
+  "default link @string                 String",
+  "default link @string.documentation   Comment",
```

Comment:
These are docstrings as in Python or Julia; not 100% happy with this link and open to better suggestions.

Reply:
Should link to `String`, imo.